qdrant-landing/content/documentation/faq/qdrant-fundamentals.md

```diff
@@ -36,11 +36,8 @@ What Qdrant can do:
 - Apply full-text filters to the vector search (i.e., perform vector search among the records with specific words or phrases)
 - Do prefix search and semantic [search-as-you-type](../../../articles/search-as-you-type/)
 - Sparse vectors, as used in [SPLADE](https://github.com/naver/splade) or similar models
-
-What Qdrant plans to introduce in the future:
-
-- ColBERT and other late-interaction models
-- Fusion of the multiple searches
+- [Multi-vectors](../../concepts/vectors/#multivectors) for ColBERT and other late-interaction models
```

Comment:
```suggestion
- [Multi-vectors](../../concepts/vectors/#multivectors), for example ColBERT and other late-interaction models
```

I think we should not make ColBERT the default use case for multivectors. It's applicable if we have multiple embeddings per object.